spirv_cross/src/msl.rs

```diff
@@ -277,6 +360,34 @@ impl spirv::Compile<Target> for spirv::Ast<Target> {
             }),
         );
 
+        self.compiler.target_data.const_samplers.clear();
+        self.compiler.target_data.const_samplers.extend(
+            options.const_samplers.iter().map(|(loc, data)| unsafe {
+                use std::mem::transmute;
+                br::MslConstSamplerMapping {
+                    desc_set: loc.desc_set,
+                    binding: loc.binding,
+                    sampler: br::SPIRV_CROSS_NAMESPACE::MSLConstexprSampler {
+                        coord: transmute(data.coord),
+                        min_filter: transmute(data.min_filter),
+                        mag_filter: transmute(data.mag_filter),
+                        mip_filter: transmute(data.mip_filter),
+                        s_address: transmute(data.s_address),
+                        t_address: transmute(data.t_address),
+                        r_address: transmute(data.r_address),
+                        compare_func: transmute(data.compare_func),
+                        border_color: transmute(data.border_color),
+                        lod_clamp_min: data.lod_clamp_min as f32 / 16.0,
```

Comment:
nit: some explanation about the `16.0` for `LodBase16` (u8) conversion to `f32` would be useful